src/components/TextInputWithLabel.js

```diff
@@ -1,80 +0,0 @@
-import _ from 'underscore';
```

Comment:
This component is not used anywhere in the App codebase anymore (search `TextInputWithLabel` to see), so I think it's safe to remove it. It doesn't make sense to keep in the future because we already have this component which will let us make TextInputs with Labels and this is redundant. Better to remove it now before someone uses it: https://github.com/Expensify/App/blob/9b0a4749e8f0d1efd3a38bf2ff89fdc086bf1fe8/src/components/TextInput/BaseTextInput.js#L162-L174